lib/notion/api/endpoints/pages.rb

```diff
@@ -38,7 +38,10 @@ def page(options = {})
         # @option options [Object] :children
         #   An optional array of Block objects representing the Page’s content
         def create_page(options = {})
-          throw ArgumentError.new('Required argument :parent.database_id missing') if options.dig(:parent, :database_id).nil?
+          unless options.dig(:parent, :database_id).present? || options.dig(:parent, :page_id).present?
```

Comment:
This raises an error as if, say, `page_id` is not present in the parent `option`, `options.dig(:parent, :page_id)` will be `nil` and `nil.present?` raises. The following does a similar check but does not raise:

```ruby
if options.dig(:parent, :database_id).nil? && options.dig(:parent, :page_id).nil?
```

Reply:
Made this change.